docs/sources/articles/configuring.md

```diff
@@ -60,39 +80,158 @@ You can start/stop/restart `docker` using
 ### Configuring Docker
 
 You configure the `docker` daemon in the `/etc/default/docker` file on your
-system.  You do this by specifying values in a `DOCKER_OPTS` variable. 
+system.  You do this by specifying values in a `DOCKER_OPTS` variable.
```

Comment:
Looks like two spaces before "You" (not your fault)

The trailing space after "variable." is intentional, so that "To configure..." is at a new line.

You can either put back the trailing space, or add a blank line before "To configure"